arch/Kconfig

```diff
@@ -349,25 +349,25 @@ config ARCH_HAS_THREAD_ABORT
 # Hidden PM feature configs which are to be selected by
 # individual SoC.
 #
-config HAS_STATE_LOW_POWER_1
+config HAS_STATE_SLEEP_1
```

Comment:
Wondering if `HAS_SLEEP_STATE_1` would work. I like it when it reads naturally in English.

Reply:
I was wondering about that myself. The reason I kept `HAS_STATE_` pattern was to make sure the names match definitions of `enum power_states` in `include/power.h`. The enum defines constants such as `SYS_POWER_STATE_SLEEP_1`, `SYS_POWER_STATE_DEEP_SLEEP_3`. They - in turn - use `SYS_POWER_STATE_` to match the enum name.

To use `HAS_SLEEP_STATE_1` I believe we should rename also the enum identifiers. It's a bigger change but yes, we could to it.

Reply:
Sounds good to me at least.